prediction_market_agent_tooling/markets/polymarket/polymarket.py

```
@@ -432,6 +437,133 @@ def get_position(
+        )
+        for prev_position in prev_positions:
+            for position_outcome, token_amount in prev_position.amounts_ot.items():
+                if position_outcome != outcome:
+                    self.sell_tokens(
+                        outcome=position_outcome,
+                        amount=token_amount,
+                        api_keys=api_keys,
+                    )
+
+    @staticmethod
+    def get_binary_market(id: str) -> "PolymarketAgentMarket":
```

Ruff (0.15.7)

[error] Function argument `id` is shadowing a Python builtin

(A002)